screens/rewards/RewardsScreen.js

```diff
@@ -208,6 +219,18 @@ const styles = StyleSheet.create({
     textAlign: 'center'
   },
   tabView: {
-    flex: 1
+    flex: 1,
```

Comment:
Looks good, but as a todo I feel like we should create a separate style-sheet or components for this screen (as is the case for the other screens).